tests/modules/nf-core/diamond/blastp/main.nf

```diff
@@ -7,22 +7,20 @@ include { DIAMOND_BLASTP } from '../../../../../modules/nf-core/diamond/blastp/m
 
 workflow test_diamond_blastp {
 
-    db = [ file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true) ]
     fasta = [ file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true) ]
     out_ext = 'txt'
     blast_columns = 'qseqid qlen'
 
-    DIAMOND_MAKEDB ( db )
-    DIAMOND_BLASTP ( [ [id:'test'], fasta ], DIAMOND_MAKEDB.out.db, out_ext, blast_columns )
+    DIAMOND_MAKEDB ( [ [id:'test'], fasta ] )
+    DIAMOND_BLASTP ( [ [id:'test'], fasta ], DIAMOND_MAKEDB.out.db.map { meta, dmnd -> dmnd }, out_ext, blast_columns )
```

Comment:
Maybe it makes more sense to add s meta2 to the dB input channel of blastp? (Even if it does nothing)

Reply:
Done, wrote test2 in the second ones

Reply:
Maybe I wasn't clear, my idea was:

1. Emit from MAKEDB the output db with meta `[meta, .dmnd]`
2. BLASTP/BLASTx `db` input channels expect a `[meta .dmnd]`, thus no channel manipulation necessary

Reply:
Oh I see, that's easy to implement. But the taxprofiler pipeline is using the DIAMOND/BLASTX module. Should I still proceed with the change?

Reply:
As discussed on slack: not an issue, that will be my problem 🤣

Reply:
Thanks, really good to know! :D